packages/sirius-web/backend/sirius-web-application/src/main/resources/schema/siriusweb.graphqls

```diff
@@ -93,7 +94,9 @@ type CreateProjectSuccessPayload {
 
 input CreateProjectFromTemplateInput {
   id: ID!
+  name: String!
   templateId: ID!
+  natures: [String!]!
```

Comment:
Honestly I would pretty much remove this attribute from both inputs. I don't see its need.

Reply:
hmm.. maybe the second statement I made can be done in the backend 🤔